.github/workflows/ci.yml

```diff
@@ -47,6 +45,7 @@ jobs:
   unit-tests:
     needs: [prep-testbed]
     runs-on: ubuntu-latest
+    if: ${{ needs.prep-testbed.outputs.matrix != '[]' }}
```

Comment:
ah so if the matrix is empty the linters will run on the entire proj dir, right?